src/Requests/Company/MemberIndexRequest.php

```diff
@@ -10,23 +10,23 @@
 class MemberIndexRequest extends Request
 {
     /**
-     * The ID of the requested company.
+     * The ID or slug of the requested company.
      *
-     * @var int
+     * @var string|int
      */
-    protected $companyId;
+    protected $companyKey;
 
     /**
      * Create a new MemberIndexRequest instance.
      *
-     * @param  int  $id
+     * @param  string|int  $id
      * @return void
      */
-    public function __construct(int $id)
+    public function __construct(string $id)
     {
         parent::__construct();
 
-        $this->companyId = $id;
+        $this->companyKey = $id;
```

Comment:
The parameter is still `$id`. :see_no_evil:

Reply:
Say what now? You saw nothing 🙃 

I blame Friday mornings